modules/apps/portal-remote/portal-remote-json-web-service-web/src/main/java/com/liferay/portal/remote/json/web/service/web/internal/helper/JSONWebServiceActionsManagerUtil.java

```diff
@@ -38,18 +41,6 @@ public static JSONWebServiceAction getJSONWebServiceAction(
 			httpServletRequest);
 	}
 
-	public static JSONWebServiceAction getJSONWebServiceAction(
```

Comment:
@julschong The commit message is saying moving this class to another place, why you are removing these methods?

Reply:
@tinatian I will separately remove these.
I removed them because this Util class is now only accessible inside our web module. and those methods are not used anymore.